charts/node-local-dns/templates/configmap.yaml

```diff
@@ -31,6 +31,31 @@ data:
         serve_stale
         {{- end }}
       }
+      {{- if $v.plugins.template }}
+      template {{ $v.plugins.template.class }} {{ $v.plugins.template.type }} {{- if $v.plugins.template.zone }} {{ $v.plugins.template.zone }} {{ end }} {
```

Comment:
Other plugins (cache, forward) are using `parameters` key to pass plugin arguments.

Reply:
changed